### PR TITLE
About repository

### LINKED ISSUE #28: Add "About Repository" section/pages
Meta-data for repositories is ignored. Data must be displayed in an aesthetically pleasing manner. 

The data displayed should include: 
- [x] Description
- [x] Back-link to the original source
- [x] License of material within
- [x] Quick link to go back to main page with filters pulled from the repository

umple-ucosp/umpr.core#18 has more on this issue from the core side.

### PR DESCRIPTION
**Pull request by Nava2**
This closes #28. 
I really dislike multi-page web applications and I don't think this has enough data to really make it worth two pages. So I have this: 
Popover

This includes the changes from #29, but the movement of the filters is irrelevant here.

### CODE REVIEW

File: app/views/files.html.php [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        <td class="col-diagram-type"><?= ucfirst(str_replace('DiagramType', '', $repo->getDiagramType())) ?></td>
+        <td class="col-input-type"><?= $file->getImportType(); ?></td>
+        <td class="col-name">
+          <?= $file->getPath() ?>
+
+          &nbsp;
+
+          <div style="float: right">
+
+          <?php if ($file->isSuccessful() || g('IMPORT_STATES')[$file->getState()] > g('IMPORT_STATES')["Fetch"]) { ?>
```
Comment (aforward) [inline app/views/files.html.php]: Invalid php 5.3

```
$import_states = g('IMPORT_STATES');
$import_states[$file->getState()]
```

File: app/views/files.html.php [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                      } ?>">
+                (Source)
+            </a>
+          <?php } else { ?>
+            <span class="text-warning" title="Unable to fetch source">(Source)</span>
+          <?php } ?>
+
+          &nbsp;
+
+          <?php if ($file->isSuccessful() || g('IMPORT_STATES')[$file->getState()] >= g('IMPORT_STATES')["Model"] ) { ?>
```
Comment (aforward) [inline app/views/files.html.php]: Fix

```
](
```

File: app/views/files.html.php [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+                    aria-controls="message-row-<?= $idTag ?>">
+              <span class="glyphicon glyphicon-remove-circle"></span>
+                <?= substr_replace($file->getState(), "", 0, 5) ?>
+              <span class="glyphicon collapse-direction" ></span>
+            </button>
+          <?php } ?>
+        </td>
+
+        <td class="col-umple-online">
+          <?php if ($file->isSuccessful() || g('IMPORT_STATES')[$file->getState()] >= g('IMPORT_STATES')["Model"] ) { ?>
```
Comment (aforward) [inline app/views/files.html.php]: Fix

```
)[
```

**Comment by aforward** [comment]
Fix the 5.3 incompatibilities and then good to go

**Comment by Nava2** [comment]
@aforward I _may_ have forgotten to merge your changes previously.

**Pull request merged**